pkg/prometheus/server/rules.go

```diff
@@ -206,7 +200,7 @@ func (c *Operator) selectRuleNamespaces(p *monitoringv1.Prometheus) ([]string, e
 func makeRulesConfigMaps(p *monitoringv1.Prometheus, ruleFiles map[string]string, opts ...operator.ObjectOption) ([]v1.ConfigMap, error) {
 	//check if none of the rule files is too large for a single ConfigMap
 	for filename, file := range ruleFiles {
-		if len(file) > maxConfigMapDataSize {
+		if len(file) > operator.MaxConfigMapDataSize {
```

Comment:
the block shouldn't be needed anymore since the operator should have called ValidateRule() before. 
```suggestion
```